main.py

```diff
@@ -45,13 +40,22 @@ def run_predict_model():
     parser.add_argument("--train_model", type=bool)
     parser.add_argument("--predict_model", type=bool)
     args = parser.parse_args()
+
+    dataset_maker = DatasetMaker(
+        input_file_name="final_taxi_data",
+        test_size=0.3,
+        save_files=True,
+        output_train_test_file_names=("train_data", "test_data"),
+        output_cleaned_file_name="train_data_clean",
+    )
+
     if args.make_dataset is True:
```

Comment:
can't it be just `if args.make_dataset:`

Reply:
True